consul/consul
# User page: new following section

What
====
Show user followable activity on public user page.

**Refactor**

* Remove popup on follow and unfollow buttons
* Add AJAX notice on follows controller actions to keep user better informed

How
===
Add new tab to show followed objects by user.

Screenshots
===========
![captura de pantalla 2017-07-13 a las 18 47 13](https://user-images.githubusercontent.com/15726/28177640-bf239fe0-67fb-11e7-97f8-d4a8f5f8522a.png)
![captura de pantalla 2017-07-13 a las 18 46 58](https://user-images.githubusercontent.com/15726/28177641-bf25a344-67fb-11e7-82a9-f5177877d772.png)

**Refactor screenshots**
![captura de pantalla 2017-07-19 a las 19 54 50](https://user-images.githubusercontent.com/15726/28381785-3e437998-6cbc-11e7-88d3-a369999e63c2.png)
![captura de pantalla 2017-07-19 a las 19 54 38](https://user-images.githubusercontent.com/15726/28381786-3e46a87a-6cbc-11e7-80e2-7b717de55339.png)


Test
====
Increased test coverage to:
- Check automatic following tab activation when filter "follows" defined
- Check "Following" tabs UI behaviour
- Check following accordion behavior
- Check followed object presence in correct accordion tab
- Check action buttons presence,  they should be shown when current user has enough rights
- Check following and unfollowing notices. 

Deployment
==========
As usual.
